packages/ccda/src/systems.ts

```diff
@@ -0,0 +1,630 @@
+  { ccdaValue: 'unknown', fhirValue: 'unknown', displayName: 'Unknown' },
+]);
+
+export const PROCEDURE_STATUS_MAPPER = new EnumMapper('ProcedureStatus', '', '', [
+  { ccdaValue: 'completed', fhirValue: 'completed', displayName: 'Completed' },
+  { ccdaValue: 'aborted', fhirValue: 'stopped', displayName: 'Stopped' },
+  { ccdaValue: 'cancelled', fhirValue: 'not-done', displayName: 'Not Done' },
+  { ccdaValue: 'new', fhirValue: 'not-done', displayName: 'Draft' },
+  { ccdaValue: 'unknown', fhirValue: 'unknown', displayName: 'Unknown' },
+]);
+
+export const MEDICATION_STATUS_MAPPER = new EnumMapper<
+  Required<MedicationRequest['status']>,
+  'active' | 'completed' | 'aborted' | 'cancelled'
+>('MedicationStatus', '', MEDICATION_REQUEST_STATUS_VALUE_SET, [
+  { ccdaValue: 'active', fhirValue: 'active', displayName: 'Active' },
+  { ccdaValue: 'completed', fhirValue: 'completed', displayName: 'Completed' },
+  { ccdaValue: 'aborted', fhirValue: 'stopped', displayName: 'Stopped' },
+  { ccdaValue: 'cancelled', fhirValue: 'cancelled', displayName: 'Cancelled' },
+  { ccdaValue: 'aborted', fhirValue: 'entered-in-error', displayName: 'Entered in Error' },
+  { ccdaValue: 'active', fhirValue: 'draft', displayName: 'Draft' },
+  { ccdaValue: 'cancelled', fhirValue: 'unknown', displayName: 'Unknown' },
+]);
+
+export const OBSERVATION_CATEGORY_MAPPER = new EnumMapper<string, string>(
+  'ObservationCategory',
+  '',
+  OBSERVATION_CATEGORY_CODE_SYSTEM,
+  [
+    // ## social-history
+    // FHIR Definition: Social History Observations define the patient's occupational, personal (e.g., lifestyle), social, familial, and environmental history and health risk factors that may impact the patient's health.
+    // Mapped C-CDA Templates:
+    // - Social History Observation (2.16.840.1.113883.10.20.22.4.38)
+    // - Social History Observation V2 (2.16.840.1.113883.10.20.22.4.38.1)
+    // - Smoking Status Observation (2.16.840.1.113883.10.20.22.4.78)
+    // - Current Smoking Status Observation (2.16.840.1.113883.10.20.22.4.78.2)
+    // - Tobacco Use Observation (2.16.840.1.113883.10.20.22.4.85)
+    // - Basic Occupation Observation (2.16.840.1.113883.10.20.22.4.503)
+    // - Basic Industry Observation (2.16.840.1.113883.10.20.22.4.504)
+    // - Sexual Orientation Observation (2.16.840.1.113883.10.20.22.4.501)
+    { ccdaValue: OID_SOCIAL_HISTORY_OBSERVATION, fhirValue: 'social-history', displayName: 'Social History' },
+    { ccdaValue: OID_SOCIAL_HISTORY_OBSERVATION_V2, fhirValue: 'social-history', displayName: 'Social History V2' },
+    { ccdaValue: OID_SMOKING_STATUS_OBSERVATION, fhirValue: 'social-history', displayName: 'Smoking Status' },
+    {
+      ccdaValue: OID_CURRENT_SMOKING_STATUS_OBSERVATION,
+      fhirValue: 'social-history',
+      displayName: 'Current Smoking Status',
+    },
+    { ccdaValue: OID_TOBACCO_USE_OBSERVATION, fhirValue: 'social-history', displayName: 'Tobacco Use' },
```

Comment:
Note to myself: The reverse would be true here, which means Observation categories are probably one area where there is a lot of loss currently